nomos-da/network/core/src/swarm/executor.rs

```diff
@@ -161,15 +162,30 @@ where
     async fn handle_behaviour_event(&mut self, event: ExecutorBehaviourEvent<Membership>) {
         match event {
             ExecutorBehaviourEvent::Sampling(event) => {
+                tracing::info!(counter.behaviour_events_received = 1, event = "sampling");
                 self.handle_sampling_event(event).await;
             }
             ExecutorBehaviourEvent::ExecutorDispersal(event) => {
+                tracing::info!(
+                    counter.behaviour_events_received = 1,
```

Comment:
Does this increment the counter? Or it resets them?

Reply:
This one resets the counter, the event is still sent to the prometheus, allowing us to query and count these events in chosen timeframe.

Reply:
Nice, I guessed thats how it works but wanted to understand it properly. Thanks!

Reply:
Oh, I actually expected this one to increase the counter. For increasing would it be `+= 1`?

Reply:
What operator do you use, then? Do you use `=`, `+=`...?

Reply:
A dedicated variable needs to be created and incremented or decremented before passing to the tracing macro

Reply:
If you guys give your ok then fine by me. That being said, I feel like the total amount of behaviours is a useful kind of metric.

Reply:
@danielSanchezQ The reason I mentioned that is because I felt that for the opentelemetry side of things you actually need to have a `Counter` object that tracks the total value.

@bacv Good to know. Will it work on the opentelemetry side of things, though?

Reply:
Clarified with @bacv by PM. Summary: Yes, this is an ad-hoc solution; but let's start simple and create proper abstractions incrementally.

Reply:
> Clarified with @bacv by PM. Summary: Yes, this is an ad-hoc solution; but let's start simple and create proper abstractions incrementally.

I've seen the same usage in other codebases, so we are not reinventing the wheel. This will be enough.

Reply:
> @bacv Good to know. Will it work on the opentelemetry side of things, though?

From my tests with prometheus at least, it works. This solution allows us to have a simpler metrics definition in the codebase. We'll definitely need more advanced meters, but for now lets use this solution. Later we'll add more helper methods for tracing/logs/metrics inside the `nomos-tracing` crate to keep rest of the codebase simpler.